genesyscloud/idp_adfs/resource_genesyscloud_idp_adfs_schema.go

```diff
@@ -0,0 +1,97 @@
+package idp_adfs
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+/*
+resource_genesycloud_idp_adfs_schema.go holds four functions within it:
+
+1.  The registration code that registers the Datasource, Resource and Exporter for the package.
+2.  The resource schema definitions for the idp_adfs resource.
+3.  The datasource schema definitions for the idp_adfs datasource.
+4.  The resource exporter configuration for the idp_adfs exporter.
+*/
+const resourceName = "genesyscloud_idp_adfs"
+
+// SetRegistrar registers all of the resources, datasources and exporters in the package
+func SetRegistrar(regInstance registrar.Registrar) {
+	regInstance.RegisterResource(resourceName, ResourceIdpAdfs())
+	regInstance.RegisterExporter(resourceName, IdpAdfsExporter())
+}
+
+// ResourceIdpAdfs registers the genesyscloud_idp_adfs resource with Terraform
+func ResourceIdpAdfs() *schema.Resource {
+	return &schema.Resource{
+		Description: `Genesys Cloud idp adfs`,
+
```

Comment:
If you look here - https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/main/genesyscloud/resource_genesyscloud_idp_adfs.go#L50 , you'll see what this description was before. I think it is better to keep it how it was because it provides more info to the end user. You'll need to run `go generate` again after making that change.